src/hist/basehist.py

```diff
@@ -13,6 +13,8 @@
 from .axis import Regular
 from .axestuple import NamedAxesTuple
 
+plt.rcParams.update({"figure.max_open_warning": 0})
```

Comment:
Let's not add this quite yet. I'll see if I can find anything to improve this. If not, then we add this.